Documentation/markdown/SetupHelp.md

```diff
@@ -50,13 +50,16 @@ These steps need to be completed before you can proceed with compiling OIP.
 	* Add this path to the environment variables. (Create new environment variable called `Boost_INCLUDE_DIR`. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. `C:\thirdparty\vs2017\x64\boost_1_65_1`).
 
 5. **Anaconda 2** (version with Python 2.7) - find [here](https://repo.anaconda.com/archive/Anaconda2-2019.10-Windows-x86_64.exe). 
-6. **PROJ 4:**
+6. **PROJ :**
 
 	*	Download OSGeo4W64 Installer (find [here](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86_64.exe)).
 	*	In Installer select *Advanced Install*
 	*	Choose to install from Internet
-	*	Select directory, where PROJ 4 will be installed (`C:\OSGeo4W64` should be default option).
-	*	Select local package directory (e.g. `C:\local`)
+	*	Select directory, where PROJ will be installed.
+
+	NOTE: CMake expects it at `C:\OSGeo4W64`, which should be the default option. If you deviate from this, please set the `PROJ_INSTALL_DIR` variable correspondingly.
```

Comment:
This is not true anymore. Please correct with the correct CMake variable name.

Reply:
Resolved: 213c48b